contributing/markup-format.md

```diff
@@ -218,9 +218,58 @@ Property    | Description
 
 ## Reusing content
 
-To reuse content in multiple places, you can save it as a shortcode
-and then use [transclusion](https://en.wikipedia.org/wiki/Transclusion) to include it elsewhere.
-For this, use the `readFile` shortcode.
+To reuse Markdown content in multiple places,
+use [transclusion](https://en.wikipedia.org/wiki/Transclusion) to include it.
+
+1. Create a new `.md` file in the `docs/layouts/shortcodes` directory.
+   Make sure the context is clear from the name, for example `reuse_markdown.md`.
+1. Fill it with the content you want to repeat.
+1. Include the file in the other locations like so:
+
+   ```markdown
+   {{% reuse_markdown %}}
+   ```
+
+### Variables in the file
+
+You can pass variables to the file:
+
+```markdown
+{{% reuse_markdown FileName="1" %}}
+```
+
+Use the variable in the file:
+
+```markdown
+The file's name is: {{ .Get "FileName" }}
+```
+
+### Inner content
+
+For longer content specific to each file, you can put content inside the shortcode:
```

Comment:
I'm not sure I fully get what this one does precisely

Reply:
Compare:
* https://github.com/platformsh/platformsh-docs/blob/main/docs/src/guides/drupal9/deploy/deploy.md?plain=1#L23 => https://docs.platform.sh/guides/drupal9/deploy/deploy.html#data-migration
* https://github.com/platformsh/platformsh-docs/blob/main/docs/src/guides/micronaut/deploy/deploy.md => https://docs.platform.sh/guides/micronaut/deploy/deploy.html#data-migration

Reply:
Okey now I get it, with `{{ .Inner | .Page.RenderString }}` you tell **where** the optional additional text gets to be added in:
```
{{% reuse_markdown %}}
additional text
{{% /reuse_markdown %}}
```